core/src/main/java/io/questdb/cairo/TxReader.java

```diff
@@ -40,7 +40,7 @@ public class TxReader implements Closeable, Mutable {
     public static final long PARTITION_SIZE_MASK = 0x80000FFFFFFFFFFFL;
     protected static final long DEFAULT_PARTITION_TIMESTAMP = 0L;
     protected static final int NONE_COL_STRUCTURE_VERSION = Integer.MIN_VALUE;
-    protected static final int PARTITION_COLUMN_VERSION_OFFSET = 3;
+    protected static final int PARTITION_FLAGS_OFFSET = 3;
```

Comment:
nit: might be a good idea to document that this offset is currently unused and kept for future purposes.

Reply:
I'm going to use it very soon